ote/src/cljs/ote/ui/table.cljs

```diff
@@ -19,6 +19,7 @@
                      label-style
                      row-style show-row-hover?
                      on-select row-selected? no-rows-message class] :as opts} headers rows]
+  (let [random-table-id (str "tid-"(rand-int 999))]
```

Comment:
Maybe (java.util.UUID/randomUUID) instead of rand-int? Lower chance of collision.

Reply:
Or id via `range` to have no chance of collision?

Reply:
For the sake of discussion, another alternative is to pass the table :key value from view who is creating the table, to here. View could assign unique keys using a string and an iterated integer.

Reply:
I leave it as it is. We have max 3 tables on same page so there is very low change of collision. We have a ticket for creating new table component and I don't want to spend any more time fixing this old one.